### PR TITLE
Funding page suggestions

### PR DESCRIPTION
**Pull request by deansallinen**

### CODE REVIEW

File: src/routes/[network]/(account)/fund/+page.svelte [hunk 1]
```diff
@@ -151,78 +153,77 @@
 	}
 </script>
 
-<div class="max-w-5xl space-y-6">
-	<h2 class="h4">{m.token_purchase_providers()}</h2>
+<Stack class="gap-12">
```
Comment (deansallinen) [inline src/routes/[network]/(account)/fund/+page.svelte:156]: add a bigger gap between sections

File: src/routes/[network]/(account)/fund/+page.svelte [hunk 2]
```diff
@@ -151,78 +153,77 @@
...
+	<Stack>
+		<h2 class="h4">{m.token_purchase_providers()}</h2>
 
-	<Grid tag="ul" class="grid-cols-1 lg:grid-cols-2 xl:grid-cols-3">
-		{#each ON_RAMP_PROVIDERS as service}
-			<Card tag="li" class="h-full w-full p-6">
-				<div>
-					<div class="mb-4 flex items-center justify-center">
-						<img src={service.logo} alt={service.id} class="h-24 w-3/5 object-contain" />
+		<MultiCard tag="ul">
```
Comment (deansallinen) [inline src/routes/[network]/(account)/fund/+page.svelte:160]: Multicard here takes card of the switch between mobile and desktop

File: src/routes/[network]/(account)/fund/+page.svelte [hunk 3]
```diff
@@ -151,78 +153,77 @@
...
+							<DLRow title={m.limits()}>
+								<DD>
+									<p class="text-white">{service.limits.daily}</p>
+								</DD>
+							</DLRow>
+							<DLRow title={m.details()}>
+								{#each service.details as detail}
+									<DD class="text-white">{detail}</DD>
+								{/each}
+							</DLRow>
```
Comment (deansallinen) [inline src/routes/[network]/(account)/fund/+page.svelte:182]: `dl` can take multiple `dd` elements per `dt` so I've done that here. Slight deviation from the design but it matches the rest of the card.

File: src/routes/[network]/(account)/fund/+page.svelte [hunk 4]
```diff
@@ -151,78 +153,77 @@
...
-						/>
-					</div>
-					<h3 class="text-center text-lg font-medium">{exchange.name}</h3>
-				</a>
-			</Card>
-		{/each}
-	</Grid>
+		<p>{m.where_eos_can_be_purchased()}</p>
+
+		<Grid tag="ul" itemWidth="10rem" class="">
```
[the remark this answers is not in the document]
Comment (dafuga) [reply src/routes/[network]/(account)/fund/+page.svelte:206]: I thought that having 2 cards per row looked fine on mobile, but having one item per row also works.

**Review by dafuga — APPROVED** [review]

**Pull request merged**